registry.json

```diff
@@ -3129,5 +3129,17 @@
       "telegram":"PeachPoolADA",
       "operatorName":"",
       "github":"amirderakh"
-   }
+   },
+    {
+ "ticker": "AAAU1",                                                       
+ "poolId":"64f6bd547b5ec3c35e18d1b09956af8dd2e53b7da7a9b7f58f3c5580",
+ "social": {                                                            
+   "twitter": "Richard79337561",
+   "facebook": "@adabankau",
+   "website":"https://adabank.com.au"
```

Comment:
The validation is failing now because "instagram" and "youtube" are required fields. This branch has no conflicts so you can just push the changes